thirdeye-server/src/main/java/ai/startree/thirdeye/auth/ResourceIdentifier.java

```diff
@@ -15,37 +15,50 @@
 
 import ai.startree.thirdeye.spi.datalayer.dto.AbstractDTO;
 import ai.startree.thirdeye.spi.datalayer.dto.AlertDTO;
+import ai.startree.thirdeye.spi.datalayer.dto.AlertTemplateDTO;
+import javax.annotation.Nullable;
 
 public class ResourceIdentifier {
 
   static public final String DefaultNamespace = "default";
 
   public final String name;
   public final String namespace;
-  public final String entityType;
+  public final EntityType entityType;
 
-  public ResourceIdentifier(final String name, final String namespace, final String entityType) {
+  public ResourceIdentifier(final String name, @Nullable final String namespace,
```

Comment:
`@Nullable` is nice. but how are we in general planning to use the namespace attribute?